testapp/exam/models.py

```diff
@@ -20,7 +21,7 @@ class Profile(models.Model):
         ("python", "Python"),
         ("bash", "Bash"),
         ("C", "C Language"),
-        ("C++", "C++ Language"),
+        ("CPP", "C++ Language"),
```

Comment:
Why are C and CPP alone upper case and the rest lower case?  Make them all lowercase.